### PR TITLE
fix-headlines

### PR DESCRIPTION
**Pull request by Tianqi-Dotes**
According to the rules below:
1. Use Headline-style capitalization in all headlines.
2. Use document name as lvl.1(#) heading.
3. Use ascend count of # to the sub-headings.
4. Use a blank line under and below each heading.
5. Delete number counters in the sub-headings. (like '1.1.x')

### CODE REVIEW

File: docs/en-us/2.0.3/user_doc/architecture/configuration.md [hunk 1]
```diff
@@ -47,12 +50,9 @@ Currently, all the configuration files are under [conf ] directory. Please check
 ├─ui                                front-end web resources directory
 ├─lib                               DS .jar dependencies directory
 ├─install.sh                        auto-setup DS services script
-
-
 ```
 
-
-# Configurations in Details
+## Configurations in Details
```
Comment (zhongjiajie) [inline docs/en-us/2.0.3/user_doc/architecture/configuration.md:55]: How about `in`

**Comment by zhongjiajie** [comment]
Good job. but should we add some lint to do it, instead of check by a human? I find our we have some lint about markdown

**Comment by zhongjiajie** [comment]
I did some research before, which you could see in https://github.com/apache/dolphinscheduler/issues/6535 , section `CI`

File: docs/en-us/2.0.3/user_doc/guide/installation/docker.md [hunk 2]
```diff
@@ -5,17 +5,17 @@
  - [Docker](https://docs.docker.com/engine/install/) 1.13.1+
  - [Docker Compose](https://docs.docker.com/compose/) 1.11.0+
 
-## How to use this Docker image
+## How to use this Docker Image
```
[the remark this answers is not in the document]
Comment (Tianqi-Dotes) [reply docs/en-us/2.0.3/user_doc/guide/installation/docker.md:8]: done

File: docs/en-us/2.0.3/user_doc/architecture/configuration.md [hunk 3]
```diff
@@ -70,7 +70,8 @@ serial number| service classification| config file|
 12|services log config files|API-service log config : logback-api.xml  <br /> master-service log config  : logback-master.xml    <br /> worker-service log config : logback-worker.xml  <br /> alert-service log config : logback-alert.xml 
 
 
-## 1.dolphinscheduler-daemon.sh [startup/shutdown DS application]
+### 1.dolphinscheduler-daemon.sh [startup/shutdown DS application]
```
[the remark this answers is not in the document]
Comment (Tianqi-Dotes) [reply docs/en-us/2.0.3/user_doc/architecture/configuration.md:73]: done remove all the numbers

**Comment by zhongjiajie** [comment]
@Tianqi-Dotes  there is one more thing I want to say. For now, you change version 2.0.3 docs. But we plan to release 2.0.5 docs in #706 . And the next release version is 3.0.0 from dev. Do you think we should directly change dev doc?

**Review by zhongjiajie — APPROVED** [review]
That's awesome, thanks 👍

**Comment by zhongjiajie** [comment]
CI still running, we can not merge untill it pass

**Pull request merged**